frontend/src/console/actions.ts

```diff
@@ -307,6 +329,57 @@ export const updateThemeFieldValue = (
   }
 };
 
+export const updateStaticContentOverridesFieldValue = (
+  instanceId: number,
+  fieldName: string,
+  value: string
+): OcimThunkAction<void> => async (dispatch, getState) => {
+  dispatch({
+    type: Types.UPDATE_INSTANCE_STATIC_CONTENT_OVERRIDES,
+    fieldName
+  });
+  try {
+    const response = await V2Api.instancesOpenedxConfigStaticContentOverrides({
+      id: String(instanceId),
+      data: {
+        [fieldName]: value
+      }
+    });
+
+    const { activeInstance } = getState().console;
+    if (activeInstance.data && activeInstance.data.id === instanceId) {
+      dispatch({
+        type: Types.UPDATE_INSTANCE_STATIC_CONTENT_OVERRIDES_SUCCESS,
+        data: response
+      });
+    }
+  } catch {
+    dispatch({
+      type: Types.UPDATE_INSTANCE_STATIC_CONTENT_OVERRIDES_FAILURE,
+      data: {
+        [fieldName]: value
+      }
+    });
+  }
+};
+
+export const updateHeroText = (
+  instanceId: number,
+  title: string,
+  subtitle: string
+): OcimThunkAction<void> => async (dispatch, getState) => {
+  dispatch({
```

Comment:
The `ActionType` is being dispatched twice, once here, and once in `updateStaticContentOverridesFieldValue`.

Can you remove this dispatch?